@seanlip: The backend tests are failing with the following log:
```
Traceback (most recent call last):
  File "/home/travis/build/oppia/oppia/core/tests/gae_suite.py", line 117, in <module>
    main()
  File "/home/travis/build/oppia/oppia/core/tests/gae_suite.py", line 99, in main
    suites = create_test_suites(test_target=parsed_args.test_target)
  File "/home/travis/build/oppia/oppia/core/tests/gae_suite.py", line 72, in create_test_suites
    if test_target else [loader.discover(
  File "/opt/python/2.7.14/lib/python2.7/unittest/loader.py", line 91, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
  File "/home/travis/build/oppia/oppia/constants_test.py", line 20, in <module>
    from core.tests import test_utils  # pylint: disable=relative-import
  File "/home/travis/build/oppia/oppia/core/tests/test_utils.py", line 29, in <module>
    from core.domain import collection_domain
  File "/home/travis/build/oppia/oppia/core/domain/collection_domain.py", line 28, in <module>
    import feconf
  File "/home/travis/build/oppia/oppia/feconf.py", line 33, in <module>
    raise Exception('DEV_MODE can\'t be true on production.')
Exception: DEV_MODE can't be true on production.
``` 
Can you ptal? Thanks!

--- a/feconf.py
+++ b/feconf.py
@@ -27,8 +27,8 @@ DEBUG = False
 
 # When DEV_MODE is true check that we are running in development environment.
 # The SERVER_SOFTWARE environment variable does not exist in Travis, hence the
-# need for an explicit default.
-if (constants.DEV_MODE and
+# need for an explicit check.
+if (constants.DEV_MODE and os.getenv('SERVER_SOFTWARE') and
         not os.getenv('SERVER_SOFTWARE', default='').startswith('Development')):
     raise Exception('DEV_MODE can\'t be true on production.')
 

